added FP for 9V battery holder 12BH611-GR

cyber-murmel opened the pull request:
This is the footprint for a fairly inexpensive 9V battery holder.

------------

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
  - [Datasheep](https://eu.mouser.com/datasheet/2/209/EPD-200766-1274481.pdf)
  - [Mouser Product Page](https://eu.mouser.com/ProductDetail/Eagle-Plastic-Devices/12BH611-GR?qs=sGAEpiMZZMsb53hqUvtGacf2JhNLyZlax9iiGvTwehY%3d)
  - ![Photo Picture](https://eu.mouser.com/images/eagleplasticdevices/hd/12BH611GR_t.jpg)
- [x] An example screenshot image is very helpful 
  - ![fp editor screenshot](https://i.imgur.com/8vtcUHT.png)
  - ![3D render screenshot](https://i.imgur.com/UFxigMQ.png)
- [x] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
  - there are none
- [x] Check the output of the Travis automated check scripts - fix any errors as required

Misca1234 commented:
Hi

How did you reach the x distance of -24.15 for the lower NPTH mounting hole?
Should it not be 24.9 (14 + 10.9) ?

The 3D name is different from the foot print name, any reason for that ?
BatteryHolder_Eagle_12BH611-GR
BatteryHolder_Bulgin_Eagle_12BH611-GR

cyber-murmel commented:
I fixed both.
The 3D name was a copy paste mistake.
I thought the lower mounting hole was on X axis in the middle of the  of the case, not in the middle of the other mounting holes. Changed it to -24.9 now.

Misca1234 commented:
great, thx
